packages/web/stores/transfer-history.tsx

```diff
@@ -422,12 +422,18 @@ const PendingTransfer: FunctionComponent<{
     if (!estimatedArrivalUnix || !progressRef.current) return;
 
     const updateTime = () => {
-      const humanizedTime = humanizeTime(dayjs.unix(estimatedArrivalUnix));
+      const date = dayjs.unix(estimatedArrivalUnix);
+      const humanizedTime = humanizeTime(date);
       if (progressRef.current) {
         // DANGER: We update the HTML directly because react-toastify is having issues while handling react state changes
-        progressRef.current.textContent = `${t("estimated")} ${
-          humanizedTime.value
-        } ${t(humanizedTime.unitTranslationKey)} ${t("remaining")}`;
+        progressRef.current.textContent =
+          date.diff(dayjs(), "seconds") < 5
+            ? t("aboutSecondsRemaining", {
+                seconds: "5 " + t("timeUnits.seconds"),
```

Comment:
Wait so we just hardcode to 5 seconds while it's still pending past estimated time?

Reply:
Yes, the idea is that users won’t see the exact count once it reaches a certain amount, allowing some flexibility for time estimate inaccuracies